LucasTerra.lua

```diff
@@ -6990,4 +6990,49 @@ function pvpsafeworld()
 	return not table.find({'Chrona', 'Morta', 'Mortera', 'Calva', 'Calvera', 'Eldera', 'Dolera', 'Inferna'}, $worldname)
 end
 
+-- @name 	usetrainer (credits to @Raphael)
+-- @desc 	Uses training statue that fits your best skill
+-- @returns bool
+function usetrainer()
+	local skillTypes = {'sword', 'axe', 'club', 'distance', 'magic'}
+
+	local skill = 'none'
+	if $vocation == 'sorcerer' or $vocation == 'druid' then
+		skill = 'magic'
+	elseif $vocation == 'paladin' then
+		skill = 'distance'
+	else
+		if $axe > $sword and $axe > $club then
+			skill = 'axe'
+		elseif $sword > $axe and $sword > $club then
+			skill = 'sword'
+		else
+			skill = 'club'
+		end
+	end

```

Comment:
Can we extract the part of the code that finds out which is your skill to its own function? I think it could be useful on different contexts.